test/inductor/test_aot_inductor.py

```diff
@@ -1063,6 +1098,7 @@ class AOTInductorTestNonABICompatibleCuda(TestCase):
     abi_compatible = False
     check_model = check_model
     check_model_with_multiple_inputs = check_model_with_multiple_inputs
+    allow_stack_allocation = False
```

Comment:
So I guess there are some gaps with the implementation here? What are the gaps?

Reply:
stack-allocating CUDA tensors does not make sense -- they need to be in GPU memory. If we want to add a test in CUDA mode with the flag on, we could do that, but the default test case should have it off IMO.

Reply:
!!! you're stack allocating the tensor data, not the tensor impl, ok, makes more sense lol